### ISSUE TITLE
Provide the path translation files in the released containers

### ISSUE THREAD

**Issue by ylep** [populse/soma-workflow#74]
If you want to use BrainVISA pipelines remotely in soma-workflow (e.g. Morphologist), you currently have to provide a path translation file like below, which is not completely straightforward to construct (even if it is described in the soma-workflow documentation). Actually, this file is generic for each major.minor version, and could be shipped along with the images:

```
de25977f-abf5-9f1c-4384-2585338cd7af   /casa/install/share/brainvisa-share-5.1
```

Also, users need to add the following line to `~/.soma-workflow.cfg`, maybe we could make it a default value?
```
PATH_TRANSLATION_FILES = brainvisa{/casa/install/<some_path>/brainvisa_path_translation}
```

**Comment by denisri** [comment]
Yes for the "brainvisa-shared" data it could be automatic. But I won't have time to do that before tomorrow...
